vta/hardware/xilinx/src/vta.cc

```diff
@@ -29,6 +29,10 @@
 
 #include "vta.h"
 
+#define PRAGMA_SUB(x) _Pragma (#x)
```

Comment:
can we move these pragma definitions in vta.h?

Reply:
sure, fixed.